packages/x-tailwindcss/src/x-tailwind-plugin/theme.ts

```diff
@@ -60,9 +60,9 @@ export default {
     }
   },
   fontFamily: {
-    primary: ['Inter', 'sans-serif'],
-    secondary: ['Lora', 'serif'],
-    auxiliary: ['Poppins', 'sans-serif'],
+    main: ['Inter', 'sans-serif'],
```

Comment:
This change is made because we decided to change the name as We did with colors.